demo/l2_challenge_fault.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Similar to challenge_fault.sh, but runs on L2!
+
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+
+export ETH_RPC_URL=http://127.0.0.1:9545/
+export BLOCK=$(cast block-number)
```

Comment:
where does this `cast` command come from?

Reply:
it's part of the foundry suite - https://getfoundry.sh/